rpc_jobs/irr_role_test.yml

```diff
@@ -85,8 +85,14 @@
     #  release.
     context:
       - kilo:
-          # Kilo EOL SHA
-          series: "97e3425871659881201106d3e7fd406dc5bd8ff3"
+          irr_flavor: "performance2-15"
```

Comment:
This flavor is consistent, so why not just define it once in the image/series axis?